app/src/main/java/org/xwiki/android/sync/activities/UserListAdapter.kt

```diff
@@ -44,12 +45,14 @@ import org.xwiki.android.sync.bean.ObjectSummary
  * @param searchResults Initial list
  */
 
-class UserListAdapter(private var searchResults: List<ObjectSummary>)
-    : RecyclerView.Adapter<UserListAdapter.ViewHolder>() {
+class UserListAdapter(
+    private var searchResults: List<ObjectSummary>,
+    private val context: Context
+) : RecyclerView.Adapter<UserListAdapter.ViewHolder>() {
 
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
-        val itemView = LayoutInflater.from(parent.context).inflate(R.layout.list_item_group, parent, false)
+        val itemView = LayoutInflater.from(parent.context).inflate(org.xwiki.android.sync.R.layout.list_item_group, parent, false)
```

Comment:
Is there some requirement here to use long form?

Reply:
Removed redundant qualifier.